### PR TITLE
updated zeros and zeros_like

### PR DESCRIPTION
**Pull request by jvishnuvardhan**
Corrected typo's, formatted, and add examples

### CODE REVIEW

File: tensorflow/python/keras/backend.py [hunk 1]
```diff
@@ -1212,24 +1212,30 @@ def zeros(shape, dtype=None, name=None):
   """Instantiates an all-zeros variable and returns it.
 
   Arguments:
-      shape: Tuple of integers, shape of returned Keras variable
-      dtype: String, data type of returned Keras variable
-      name: String, name of returned Keras variable
+      shape: Tuple or list of integers, shape of returned Keras variable
+      dtype: data type of returned Keras variable
+      name : name of returned Keras variable
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/backend.py]: I don't think pylint will approve of the extra space between `name` and `:`

File: tensorflow/python/keras/backend.py [hunk 2]
```diff
@@ -1212,24 +1212,30 @@ def zeros(shape, dtype=None, name=None):
...
-      >>> from keras import backend as K
-      >>> kvar = K.zeros((3,4))
-      >>> K.eval(kvar)
-      array([[ 0.,  0.,  0.,  0.],
-             [ 0.,  0.,  0.,  0.],
-             [ 0.,  0.,  0.,  0.]], dtype=float32)
+  from tensorflow.keras import backend as K
+  kvar = K.zeros((3,4))
+  K.eval(kvar)
+  array([[ 0.,  0.,  0.,  0.],
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/backend.py]: There is no longer a separation between inputs and outputs here

**Review by mihaimaruseac — COMMENTED** [review]
Looks good, but there is a minor nit (re spacing) and an issue (re formatting of inputs and outputs)

File: tensorflow/python/keras/backend.py (hunk 1 above)
Comment (jvishnuvardhan) [reply tensorflow/python/keras/backend.py]: Thanks @mihaimaruseac. I have updated it.

File: tensorflow/python/keras/backend.py (hunk 2 above)
Comment (jvishnuvardhan) [reply tensorflow/python/keras/backend.py]: I didn't quite understand your comment here. But I modified it little bit. Please check and let me know whether my changes are fine or not. Thanks!

File: tensorflow/python/keras/backend.py [hunk 3]
```diff
@@ -1212,24 +1212,29 @@ def zeros(shape, dtype=None, name=None):
...
+      dtype: data type of returned Keras variable
+      name: name of returned Keras variable
 
   Returns:
       A variable (including Keras metadata), filled with `0.0`.
       Note that if `shape` was symbolic, we cannot return a variable,
       and will return a dynamically-shaped tensor instead.
 
-  Example:
+  For Example:
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/backend.py]: This should be kept as `Example:`, afaik

File: tensorflow/python/keras/backend.py [hunk 4]
```diff
@@ -1312,22 +1317,23 @@ def zeros_like(x, dtype=None, name=None):
 
   Arguments:
       x: Keras variable or Keras tensor.
-      dtype: String, dtype of returned Keras variable.
-           None uses the dtype of x.
-      name: String, name for the variable to create.
+      dtype: dtype of returned Keras variable.
+             None uses the dtype of `x`.
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/backend.py]: `None`, in backquotes

**Review by mihaimaruseac — COMMENTED** [review]
Two more nits, but looks good

File: tensorflow/python/keras/backend.py (hunk 4 above)
Comment (jvishnuvardhan) [reply tensorflow/python/keras/backend.py]: Updated. Thanks for the suggestions!

File: tensorflow/python/keras/backend.py (hunk 3 above)
Comment (jvishnuvardhan) [reply tensorflow/python/keras/backend.py]: Updated. Thanks for the suggestions!

**Review by mihaimaruseac — APPROVED** [review]
Thank you

**Pull request merged**